Gemfile

```diff
@@ -51,3 +51,9 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+gem 'minitest-rails'
```

Comment:
This could go in the `group :test` directive. That indicates which environments to bundle the gem with. In large projects this saves space and time while deploying.